olive/systems/system_config.py

```diff
@@ -15,6 +15,16 @@
 from olive.systems.common import AzureMLDockerConfig, LocalDockerConfig, SystemType
 
 
+class VCConfig(ConfigBase):
+    instance_type: str
+    interactive: bool = False
+    imageVersion: str = ""
+    slaTier: str = "Standard"
+    priority: str = "Medium"
+    tensorboardLogDirectory: str = "/scratch/tensorboard_logs"
```

Comment:
make the naming consistent by all use snake case(variable_part) instead of came case?

Reply:
I updated the PR and removed all other configs.